lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 227 refactor onafhankelijke fractie

## Description
This PR solves a few different problems, if no onafhankelijke fractie exists, one is created when opening a form that contains a fractie selector. When selecting fracties, only one fractie that is of the onafhankelijke fractietype is shown. The correct bestuursorganen-in-de-tijd are provided to the component, so this new fractie can be attached to the correct bestuursorganen in de tijd.

## Testing
Testing can happen in a few different places, Gemeente Aarschot is a good option, since it does not contain a fractie with the name or type onafhankelijk

1. Go to the fractie route under bestuursorganen, to check which fracties actually exist
2. Go to a mandataris and open the corrigeer fouten or update state form (best to check both)
3. If you go to the fractie-selector you should see a new onafhankelijke fractie (if you look in the ember debugger under data, you can see this fractie is actually of the type onafhankelijk)
4. If you go back to the fractie route you see this fractie is added here as well

If you want to check the fact that only one fractie of the onafhankelijke fractietype is shown, you should first test this and afterwards run the migration included in this branch ben/lmb-369-migration-onafhankelijke-fracties this would result in two fracties of the onafhankelijke type in e.g Gemeente Aalst, which you can check in the fractie route, but which is not shown in the mandataris form.
